suite-native/device/src/hooks/useDetectDeviceError.tsx

```diff
@@ -37,6 +57,8 @@ export const useDetectDeviceError = () => {
 
     const isDeviceFirmwareSupported = useSelector(selectIsDeviceFirmwareSupported);
 
+    const hasDeviceError = selectedDevice?.error !== undefined;
```

Comment:
nit: `!!selectedDevice?.error`

Reply:
Also please create a selector and also use the returned value in useEffect deps

Reply:
10aacf0fcc